CMakeLists.txt

```diff
@@ -28,8 +28,8 @@
 # The SPIR-V headers from the SPIR-V Registry
 # https://www.khronos.org/registry/spir-v/
 #
-cmake_minimum_required(VERSION 3.0)
-project(SPIRV-Headers VERSION 1.5.5)
+cmake_minimum_required(VERSION 3.14)
+project(SPIRV-Headers VERSION 1.6.1)
```

Comment:
I was told by @jeremy-lunarg and @greg-lunarg that the project should come from `spirv.core.grammar.json`